great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -206,9 +245,13 @@ def _get(self, key: Tuple[str, ...]) -> ResponsePayload:  # type: ignore[overrid
             raise StoreBackendError(
                 f"Unable to get object in GE Cloud Store Backend: {jsonError}"
             )
-        except (requests.HTTPError, requests.Timeout) as http_exc:
+        except requests.HTTPError as http_err:
             raise StoreBackendError(
-                f"Unable to get object in GE Cloud Store Backend: {http_exc}"
+                f"Unable to get object in GE Cloud Store Backend: {_get_user_friendly_error_message(http_err)}"
+            )
+        except requests.Timeout as timeout_exc:
+            raise StoreBackendError(
+                f"Unable to get object in GE Cloud Store Backend: {timeout_exc}"
```

Comment:
Myabe "Unable to get object in GE Cloud Store Backend. This is likely a transient error. Please try again."

Reply:
Added.